DotNetCliVersion.txt

```diff
@@ -1 +1 @@
-2.0.0
\ No newline at end of file
+2.1.302
```

Comment:
this value is not picked up by the sh script right? As you hardcoded it there.

Reply:
This version is used by the `.cmd` and `.sh` files. Ideally, I would consolidate the build scripts into a single script.

Reply:
So is the hardcoded version string here needed? https://github.com/Microsoft/xunit-performance/pull/271/files#diff-ba04653bd26665aa2b926dbe2b144e61R21

Reply:
Good call, it needs to be updated.